internal/pkg/peer/blocksprovider/bft_header_receiver.go

```diff
@@ -110,15 +110,21 @@ func (hr *BFTHeaderReceiver) DeliverHeaders() {
 
 		case *orderer.DeliverResponse_Block:
 			blockNum := t.Block.Header.Number
-			err := hr.updatableBlockVerifier.VerifyBlockAttestation(t.Block)
-			if err != nil {
-				hr.logger.Warningf("[%s][%s] Last block verification failed, blockNum [%d], err: %s", hr.chainID, hr.endpoint, blockNum, err)
-				return
-			}
 
-			if protoutil.IsConfigBlock(t.Block) { // blocks with block.Data==nil return false
+			if !protoutil.IsConfigBlock(t.Block) { // normal blocks with block.Data==nil
```

Comment:
`IsConfigBlock` will react fast (with false) for if data==nil, so it is as cheap as testing the data directly... We optimized for that in a previous PR.

Note that we must call IsConfigBlock in any case before we try to update the config. 
I use `if else` here in order to avoid invoking `VerifyBlockAttestation` and `VerifyBlock` when a config block does arrive.